bin/game.rb

```diff
@@ -3,19 +3,34 @@
 class Game < Item
   attr_accessor :multiplayer, :last_played_at
 
-  def initialize(publish_date, multiplayer, last_played_at)
-    super(publish_date)
+  def self.from_hash(data)
+    new(data['publish_date'], data['multiplayer'], data['last_played_at'], data['id'])
+  end
+
+  def initialize(publish_date, multiplayer, last_played_at, id = nil)
+    super(publish_date, id)
+
     @multiplayer = multiplayer
     @last_played_at = Time.parse(last_played_at)
   end
 
+  def to_s
+    "Game: (#{@id}) publish date: #{@publish_date.to_date} - multiplayer: #{@multiplayer} - last payed at: #{@last_played_at.to_date}"
```

Comment:
This line has more than 120 characters, please split it in several strings, maybe like this:

```ruby
game = "Game (#{...})"
publish_date = "Publish date: #{...}"
multiplayer = "Multiplayer: #{...}"
last_played = "Last played: #{...}"

[game, publish_date, multiplayer, last_played].join(' - ')
```